opaque_keys/edx/keys.py

```diff
@@ -77,6 +77,26 @@ def block_type(self):  # pragma: no cover
         raise NotImplementedError()
 
 
+class AggregateCourseKey(OpaqueKey):
+    """
+    An :class:`opaque_keys.OpaqueKey` identifying a
+    serialized Course Key object.
+    """
+    KEY_TYPE = 'aggregate_course_key'
```

Comment:
I have change the `CANONICAL_NAMESPACE` of `AggregateCourseLocator` to `course-v2`. Now the aggregate course key format would be `course-v2:org+course`
```python
AggregateCourseKey.from_string('course-v2:edX+DemoX')
```